### PR TITLE
feat:[PagoPa-1147] AMEX filter

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- added management for AMEX payments

#### Motivation and Context

- new business request

#### How Has This Been Tested?

Junit and manual test

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by alessio-acitelli** [comment]
update_code

File: src/main/resources/application-dev.properties [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+logging.level.it.gov.pagopa=DEBUG
+logging.level.com.azure.cosmos.implementation.SqlQuerySpecLogger=TRACE
+
+# Cosmos account config
+azure.cosmos.uri=https://pagopa-d-weu-afm-marketplace-cosmos-account.documents.azure.com:443/
+azure.cosmos.key=EWulzSYxAAxbxIaPxrK0zA7WLXoTOes9eL1WyZVo7fDoQb0AXfHKoKvLr99hp3PCvNQJNX3KEmcSFQVrh1yk1A==
+azure.cosmos.database=db
+
+# Storage account config
+azure.storage.connection=DefaultEndpointsProtocol=https;AccountName=pagopadweuafmsa;AccountKey=RRDygFmNKKHpX+icQN4F3UdNzaoCgbbS4K8QDjFq0vJUCUKPpBv2DykLYm22OlqsIbZCPYNQJdEO+AStoX9jzw==;EndpointSuffix=core.windows.net
```
Comment (pasqualespica) [inline src/main/resources/application-dev.properties:16]: remove it

**Review by pasqualespica — CHANGES_REQUESTED** [review]
ok, but remove key

File: src/main/resources/application-dev.properties (hunk 1 above)
Comment (alessio-acitelli) [reply src/main/resources/application-dev.properties]: done

Comment (alessio-acitelli) [reply src/main/resources/application-dev.properties]: done

**Pull request merged**